src/Extensions/FileSubsiteSearchExtension.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Class FileSubsiteExtension|Firesphere\SolrSearch\Extensions\FileSubsiteExtension
+ *
+ * @package Firesphere\Solr
+ * @author Signify Ltd <info@signify.co.nz>
+ * @copyright Copyright (c) 2024 - now() Signify Ltd
```

Comment:
I suggest **not** using the cutesy `2024 - now()`. Lawyers are not known for their appreciation of cuteness.

I think this is sufficient:

```
@copyright Copyright 2024 Signify Ltd
```